src/Acme.UI/Views/Authentication/_Login.cshtml

```diff
@@ -0,0 +1,23 @@
+﻿@inherits Umbraco.Web.Mvc.UmbracoViewPage<Acme.UI.Models.Authentication.LoginModel>
```

Comment:
Please dont spend much time on it but put some of the [bootstrap form elements around](https://getbootstrap.com/docs/4.0/components/forms/) it just so it lays out better.